specification/kubernetesruntime/KubernetesRuntime.Management/loadbalancer.tsp

```diff
@@ -0,0 +1,65 @@
+import "./common.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.ResourceManager;
+
+namespace Microsoft.KubernetesRuntime;
+
+@doc("A LoadBalancer resource")
+model LoadBalancer is ExtensionResource<LoadBalancerProperties> {
```

Comment:
>ExtensionResource [](http://example.com/codeflow?start=22&length=17)

What all types can it extend? You should try to document the expected usage scenarios